### DISCUSSION TITLE
Got Apple Studio Display and uninstalled MonitorControl, brightness controls not working

### DISCUSSION THREAD

**Discussion by carl4286** [MonitorControl/MonitorControl#1582] [General]
I recently replaced my LG monitor (for which I originally installed MonitorControl) with an Apple Studio Display which, to my knowledge, allows brightness control natively through the keyboard controls. I uninstalled MonitorControl, but the native brightness control does not function. Is there an extra step needed to restore the default function of the brightness keys to control an Apple display? Thanks.

**Comment by waydabber** [comment]
There should not be any extra steps.
